pkg/kubelet/cm/dra/manager.go

```diff
@@ -48,6 +47,7 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	kubeletmetrics "k8s.io/kubernetes/pkg/kubelet/metrics"
 	"k8s.io/kubernetes/pkg/kubelet/pluginmanager/cache"
+	"k8s.io/kubernetes/pkg/scheduler/util"
```

Comment:
Nit:
```suggestion
	schedutil "k8s.io/kubernetes/pkg/scheduler/util"
```

For consistency and to avoid future name conflicts. "util" is too generic.

Reply:
changed to schedutil